aten/src/ATen/test/tensor_iterator_test.cpp

```diff
@@ -45,9 +45,9 @@ TEST(TensorIteratorTest, MixedDevices) {
 
 Tensor random_tensor_for_type(at::ScalarType scalar_type) {
   if (at::isFloatingType(scalar_type)) {
-    return at::randn({5, 5}, kCPU);
+    return at::randn({5, 5}, kCPU).to(scalar_type);
```

Comment:
Can't this just be `at::randn({5, 5}, TensorOptions().dtype(scalar_type)` ?

Reply:
Changed